src/main/java/de/abstractolotl/azplace/model/view/ProfileView.java

```diff
@@ -18,12 +22,17 @@ public class ProfileView {
     @JsonProperty("timestamp_registered")
     private long timestampRegistered;
 
+    private List<String> roles;
+
     @JsonProperty("user_settings")
     private UserSettings userSettings;
 
     public static ProfileView fromUser(User user) {
+        List<String> roles = Arrays.asList(user.getRoleArray());
+        roles.remove("anonymous");
```

Comment:
why remove this role?

Reply:
This role is no real role it is a fake role which is only used for usersettings